### DISCUSSION TITLE
Button Links cannot use relationship field

### DISCUSSION THREAD

**Discussion by altezza04** [directus/directus#5947] [Feature Requests]
Button Links interface, the URL field does not seem to pick up relationship field

for example, menu is a o2m field 

This work
https://localhost:8080/{{ menu }}

This doesn't work
https://localhost:8080/{{ menu.slug }}

**Comment by azrikahar** [comment]
Related discussions: #11255

---

Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
